src/build-remote/build-remote.cc

```diff
@@ -162,7 +163,15 @@ static int _main(int argc, char * * argv)
                             bestLoad = load;
                             bestSlotLock = std::move(free);
                             bestMachine = &m;
+                        } else {
+                            debug("skipping '%s': there is a better machine", m.storeUri);
```

Comment:
What does better mean?

Reply:
actually I'm not sure either, but I'm guessing it's based on the parameters above a machine that has a higher speed factor and is available right now